index.html

```diff
@@ -40,1559 +40,1614 @@
+                                placeholder="First Name"
+                                minlength="1"
+                                required
+                                onkeypress="return (event.charCode > 64 &&
+                                   event.charCode < 91) ||
+                                    (event.charCode > 96 &&
+                                     event.charCode < 123)"
+                        />
+                        <label for="firstNameCreate">First Name</label>
+                    </div>
+                    <div class="form-floating mb-3">
+                        <input
+                                name="lastName"
+                                type="text"
+                                class="form-control"
+                                id="lastNameEdit"
+                                placeholder="Last Name"
+                                required
+                                onkeypress="return (event.charCode > 64 &&
+                                event.charCode < 91) ||
+                                 (event.charCode > 96 &&
+                                  event.charCode < 123)"
+                        />
+                        <label for="lastNameEdit">Last Name</label>
+                    </div>
+                    <div class="form-floating mb-3">
+                        <select
+                                name="role"
+                                class="form-select"
+                                id="roleEdit"
+                                required
+                        >
+                            <option selected disabled value="">Choose...</option>
+                            <option value="Goalkeeper">Goalkeeper</option>
+                            <option value="Sweeper">Sweeper</option>
+                            <option value="Defender">Defender</option>
+                            <option value="Midfielder">Midfielder</option>
+                            <option value="Attacker">Attacker</option>
+                        </select>
+                        <label for="roleEdit">Role</label>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button
+                            type="button"
+                            class="btn btn-secondary"
+                            data-bs-dismiss="modal"
+                    >
+                        Close
+                    </button>
+                    <button
+                            type="submit"
+                            class="btn btn-primary"
+                            id="submitEdit"
+                            data-bs-dismiss="modal"
+                            style="background-color: #027dab"
+                    >
+                        savePlayer
+                    </button>
+                </div>
+            </form>
+
         </div>
-      </div>
     </div>
+</div>
+</div>
```

Comment:
Der er en div for meget, du må lige se hvorfor den er der eller om den bare skal væk